packages/next/src/server/route-modules/app-page/module.ts

```diff
@@ -19,12 +19,15 @@ import type { ServerComponentsHmrCache } from '../../response-cache'
 import type { FallbackRouteParams } from '../../request/fallback-params'
 
 let vendoredReactRSC
-let vendoredReactSSR
+// TODO: Ship react-dom/server.edge types
+let vendoredReactSSR: any
```

Comment:
Otherwise we end up referencing `react-dom/server.edge` in the published declaration files. This basically just restores the status quo until https://github.com/DefinitelyTyped/DefinitelyTyped/pull/72926 lands